docs/setup_installation/on_prem/hopsworks_installer.md

```diff
@@ -1,167 +1,17 @@
-# Hopsworks Installer
+# Hopsworks On premises
 
+It is possible to us Hopsworks on-premises, which means that companies can run their machine learning workloads on their own hardware and infrastructure, rather than relying on a cloud provider. This can provide greater flexibility, control, and cost savings, as well as enabling companies to meet specific compliance and security requirements.
```

Comment:
```suggestion
It is possible to use Hopsworks on-premises, which means that companies can run their machine learning workloads on their own hardware and infrastructure, rather than relying on a cloud provider. This can provide greater flexibility, control, and cost savings, as well as enabling companies to meet specific compliance and security requirements.
```